src/test/java/kitchenpos/application/OrderServiceTest.java

```diff
@@ -0,0 +1,512 @@
+
+    private OrderTableService orderTableService;
+
+    private MenuGroup menuGroup;
+
+    @BeforeEach
+    void setUp() {
+        menuGroupService = new MenuGroupService(menuGroupRepository);
+        productService = new ProductService(productRepository, menuRepository, purgomalumClient);
+        menuService = new MenuService(menuRepository, menuGroupRepository, productRepository,
+                purgomalumClient);
+        orderService = new OrderService(orderRepository, menuRepository, orderTableRepository,
+                kitchenridersClient);
+        orderTableService = new OrderTableService(orderTableRepository, orderRepository);
+        menuGroup = menuGroupService.create(MenuGroupFixture.createRequest("치킨"));
+    }
+
+
+    @Test
+    void 주문_하나에_여러_메뉴를_주문할_수_있다() {
+        Order createRequest = OrderFixture.createTakeOutRequest(
+                OrderFixture.createOrderLineItem(createFriedMenu(), 2),
+                OrderFixture.createOrderLineItem(createSeasonedMenu(), 1));
+
+        Order actual = orderService.create(createRequest);
+
+        assertThat(actual.getId()).isNotNull();
+    }
+
+    @Test
+    void 노출되지_않은_메뉴를_주문하면_예외를_던진다() {
+        Menu friedMenu = createFriedMenu();
+        menuService.hide(friedMenu.getId());
+        Order createRequest = OrderFixture.createTakeOutRequest(
+                OrderFixture.createOrderLineItem(friedMenu, 2));
+
+        assertThatThrownBy(() -> orderService.create(createRequest)).isInstanceOf(
+                IllegalStateException.class);
+    }
+
+    @Test
+    void 메뉴를_1개이상_주문하지_않으면_예외를_던진다() {
+        Order createRequest = OrderFixture.createTakeOutRequest();
+
+        assertThatThrownBy(() -> orderService.create(createRequest)).isInstanceOf(
+                IllegalArgumentException.class);
+    }
+
+    @Test
+    void 모든_주문_목록을_볼_수_있다() {
+        Order takeOutRequest = OrderFixture.createTakeOutRequest(
+                OrderFixture.createOrderLineItem(createFriedMenu(), 2));
+        Order deliveryRequest = OrderFixture.createDeliveryRequest("주소",
+                OrderFixture.createOrderLineItem(createFriedMenu(), 2));
+        orderService.create(takeOutRequest);
+        orderService.create(deliveryRequest);
+
+        List<Order> actual = orderService.findAll();
+        assertThat(actual).hasSize(2);
+    }
+
+    @Nested
+    class DeliveryOrder {
+
+        @Test
+        void 배달주문_생성시_주소를_빠뜨리면_예외를_던진다() {
+            Order createRequest = OrderFixture.createDeliveryRequest(null,
+                    OrderFixture.createOrderLineItem(createFriedMenu(), 2));
+
+            assertThatThrownBy(() -> orderService.create(createRequest)).isInstanceOf(
+                    IllegalArgumentException.class);
+        }
+
+        @Test
+        void 배달주문이_생성되면_주문은_승인을_WAITING하는_상태가_된다() {
+            Order createRequest = OrderFixture.createDeliveryRequest("주소",
+                    OrderFixture.createOrderLineItem(createFriedMenu(), 2));
```

Comment:
```suggestion
            Order createRequest = OrderFixture.배달주문_WAITING());
```
Fixture Method를 활용하여 테스트 객체를 생성하는것도 좋은 방법이지만 자주 사용되거나 의미를 가지는 Test객체 자체를 Fixture로 정의하여 가독성을 높일수 있을것 같아요 👍 
Fixture를 적용할수 있는 다른 테스트 코드도 살펴봐주세요 !

Reply:
자주 사용되는 객체를 아예 Fixture에 정의를 해버리면 더욱 편하겠네요~ 수정해보겠습니다!